### PR TITLE
Additional Resources page

### PR DESCRIPTION
**Pull request by hadarhg**
- [ ] Update descriptions for each section
- [ ] Add subtitle/description for each resource

### CODE REVIEW

**Comment by hadarhg** [comment]
R: @davorbonaci

File: src/documentation/resources.md [hunk 1]
```diff
@@ -6,11 +6,109 @@ redirect_from: /learn/resources/
 ---
 # Additional Resources for Learning about Apache Beam
 
-This page is under construction ([BEAM-509](https://issues.apache.org/jira/browse/BEAM-509)).
+This page provides links to some of our favorite videos, podcasts, and articles that will help get you started.
```
Comment (davorbonaci) [inline src/documentation/resources.md]: or learn more about Apache Beam.

File: src/documentation/resources.md [hunk 2]
```diff
@@ -6,11 +6,109 @@ redirect_from: /learn/resources/
...
+## General
+
+The following resources provide general overviews and fundamentals of Apache Beam.
+
+
+### Fundamentals of Stream Processing with Apache Beam
+
+Data Science Summit, 2016
+
+Presented by Tyler Akidau
```
Comment (davorbonaci) [inline src/documentation/resources.md]: , Apache Beam PPMC member

File: src/documentation/resources.md [hunk 3]
```diff
@@ -6,11 +6,109 @@ redirect_from: /learn/resources/
...
+Presented by Tyler Akidau 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/V35MwYcXEX0" frameborder="0" allowfullscreen></iframe>
+<br>
+
+### Apache Beam: A Unified Model for Batch and Streaming Data Processing
+
+Hadoop Summit, 2016 
+
+Presented by Davor Bonacci
```
Comment (davorbonaci) [inline src/documentation/resources.md]: , Apache Beam PPMC member

File: src/documentation/resources.md [hunk 4]
```diff
@@ -6,11 +6,109 @@ redirect_from: /learn/resources/
...
-* [Dataflow/Beam & Spark: A Programming Model Comparison](https://cloud.google.com/dataflow/blog/dataflow-beam-and-spark-comparison)
-* [Dataflow and open source - proposal to join the Apache Incubator](http://googlecloudplatform.blogspot.com/2016/01/Dataflow-and-open-source-proposal-to-join-the-Apache-Incubator.html)
+## General
+
+The following resources provide general overviews and fundamentals of Apache Beam.
+
+
+### Fundamentals of Stream Processing with Apache Beam
+
+Data Science Summit, 2016
```
Comment (davorbonaci) [inline src/documentation/resources.md]: name, location, year

File: src/documentation/resources.md [hunk 5]
```diff
@@ -6,11 +6,109 @@ redirect_from: /learn/resources/
...
+<iframe width="560" height="315" src="https://www.youtube.com/embed/fc-YigLn_gs" frameborder="0" allowfullscreen></iframe>
+<br>
+
+## Technical Details
+
+The following resources provide detailed explanations about technical concepts in Apache Beam.
+
+### Watermarks: Time and Progress in Apache Beam and Beyond
+
+Strata and Hadoop, 2016
```
Comment (davorbonaci) [inline src/documentation/resources.md]: Strata + Hadoop World, city, year

File: src/documentation/resources.md [hunk 6]
```diff
@@ -6,11 +6,109 @@ redirect_from: /learn/resources/
...
+
+Presented by Slava Chernyak
```
Comment (davorbonaci) [inline src/documentation/resources.md]: , Software Engineer at Google

**Pull request closed without merging**